.github/workflows/dispatch.yaml

```diff
@@ -0,0 +1,43 @@
+name: Update GPTScript Version
+on:
+  repository_dispatch:
+    types: release
+
+jobs:
+  update-gptscript-dep:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install jq
+        uses: dcarbone/install-jq-action@v2.1.0
+      - name: Update GPTScript Version
+        run: |
+          jq '.version = "${{ github.event.client_payload.tag }}"' package.json > temp.json && mv temp.json package.json
+          sed -i 's/version: "v[0-9.]*"/version: "${{ github.event.client_payload.tag }}"/' scripts/install-binary.js
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 21
+      - name: Update Lock
+        run: npm i --package-lock-only
+      - uses: stefanzweifel/git-auto-commit-action@v5
+        with:
+          commit_message: Automated GPTScript Version Update
+          file_pattern: 'package*.json src/install-binary.js'
+  tag-release:
```

Comment:
There is already a `release.yaml` for releasing. Do we want to combine these or get rid of one? I think we need both so that we can do one-off releases, but curious to your thoughts on how to handle that.

Reply:
`release.yaml` does the actual github 'release' and publish to npm. This job just programmatically creates the tag that kicks off that workflow in the case that gptscript was released. I think it makes sense to keep both.